community/contribute/cloud-resources/cloud-resources.md

```diff
@@ -43,18 +43,19 @@ Service users will have their default_project_id set to a specific project and w
 
 ### Projects
 
-| Unique Identifier | Project Name | Community Contact | Description | Needed until |
-|:---:|---|---|---|:---:|
-| 2237c767cf5f456da19359ed31c1c16b | p500924-scs-community | fkr | SCS Community Project | ∞ |
-| e7ee61b62ecc417199b154ae4218f25f | p500924-beermann | tibeer | Project of Tim Beermann, OSISM | ∞ |
-| b43cfafbcf1f4eb08865b2886c29e09b | p500924-cluster-api-session | garloff | cluster-api hands on session | ∞ |
-| 9b7a73e516be4cd1acbd63d543985c52 | p500924-gonicus-dev | o-otte | GONICUS GmbH | ∞ |
-| 3829cc7c8f034fc985f5055a1df6f247 | p500924-scs-healthmonitor | garloff | SCS Health Monitor | ∞ |
-| b97d38bf128b4479981c4dbe2ef70cd5 | p500924-SIG-IAM | fkr | SIG IAM | ∞ |
-| 9de7d8dc2d674e52be44904d6b338b0b | p500924-cloudmon | costelter | CloudMon Test Project | 31.12.2023 |
-| 2c9e0e4ef8d44c36807df50b06b3c81d | p500924-cloudmon-target | costelter | Target project for CloudMon tests | 31.12.2023 |
-| 3501db829014406884990a1016f3e25d | p500924-sig-monitoring1 | fkr | SIG Monitoring - cloudmon target | 31.12.2023 |
-| 602778bad3d3470cbe58c4f7611e8eb7 | p500924-dnation | chess-knight | dNation Dev Project | ∞ |
-|  |  |  |  |  |
+|        Unique Identifier         | Project Name                | Community Contact | Description                       | Needed until |
+|:--------------------------------:|-----------------------------|-------------------|-----------------------------------|:------------:|
+| 2237c767cf5f456da19359ed31c1c16b | p500924-scs-community       | fkr               | SCS Community Project             |      ∞       |
+| e7ee61b62ecc417199b154ae4218f25f | p500924-beermann            | tibeer            | Project of Tim Beermann, OSISM    |      ∞       |
+| b43cfafbcf1f4eb08865b2886c29e09b | p500924-cluster-api-session | garloff           | cluster-api hands on session      |      ∞       |
+| 9b7a73e516be4cd1acbd63d543985c52 | p500924-gonicus-dev         | o-otte            | GONICUS GmbH                      |      ∞       |
+| 3829cc7c8f034fc985f5055a1df6f247 | p500924-scs-healthmonitor   | garloff           | SCS Health Monitor                |      ∞       |
+| b97d38bf128b4479981c4dbe2ef70cd5 | p500924-SIG-IAM             | fkr               | SIG IAM                           |      ∞       |
+| 9de7d8dc2d674e52be44904d6b338b0b | p500924-cloudmon            | costelter         | CloudMon Test Project             |  31.12.2023  |
+| 2c9e0e4ef8d44c36807df50b06b3c81d | p500924-cloudmon-target     | costelter         | Target project for CloudMon tests |  31.12.2023  |
+| 3501db829014406884990a1016f3e25d | p500924-sig-monitoring1     | fkr               | SIG Monitoring - cloudmon target  |  31.12.2023  |
+| 602778bad3d3470cbe58c4f7611e8eb7 | p500924-dnation             | chess-knight      | dNation Dev Project               |      ∞       |
+| e2d358c738cc4b67872d1c9919f2468e | p500924-metering            | cah-link          | Metering Dev Project              |  31.12.2023  |
```

Comment:
@cah-link project has been spawned and you should have received an email with login credentials

please swap the Unique Identifier to 93956190702b4a7d8a8886806d57713f as this is mainly just for plusserver internal documentation, I will amend to the documentation that it can be left out when opening the PR